Core/src/Geometry/CuboidVolumeBuilder.cpp

```diff
@@ -169,6 +169,9 @@ std::shared_ptr<Acts::TrackingVolume> Acts::CuboidVolumeBuilder::buildVolume(
 
     LayerConfig lCfg;
     lCfg.surfaceCfg = {sCfg};
+    lCfg.envelopeX = {0.1 * UnitConstants::mm, 0.1 * UnitConstants::mm};
+    lCfg.envelopeY = {0.1 * UnitConstants::mm, 0.1 * UnitConstants::mm};
+    lCfg.envelopeZ = {0.1 * UnitConstants::mm, 0.1 * UnitConstants::mm};
```

Comment:
Why were these values chosen?

Reply:
mostly random. they need to be big enough to not collide with the surface tolerance but small enough to not collide with the geometry